app/views/pages/contact.html.erb

```diff
@@ -1,6 +1,28 @@
 <h1 class="govuk-heading-l govuk-!-margin-bottom-5">Contact</h1>
 
 <div class="govuk-grid-row dashboard-row">
+
   <div class="govuk-grid-column-two-thirds">
+    <%= form_tag(contact_path, method: :post, id: "contact_form") do %>
+      <div class="govuk-form-group">
+        <label class="govuk-label" for="more-detail">
+          You can use this form to ask a question, report a problem or suggest an improvement.
+        </label>
+        <span id="more-detail-hint" class="govuk-hint">
+          Do not include personal or financial information, like your National Insurance number or credit card details.
+        </span>
+        <% if @contact.present? %>
+          <% if @contact.errors[:more_detail].present? %>
+            <span class="govuk-error-message">
+              The text box
+              <%= @contact.errors[:more_detail].first %>
+            </span>
+          <% end %>
+        <% end %>
+        <textarea class="govuk-textarea" id="more-detail" name="more_detail" rows="10" aria-describedby="more-detail-hint"></textarea>
+      </div>
+
+      <button class="gem-c-button govuk-button" type="submit">Submit</button>
```

Comment:
@theKHutDeveloper - haven't come across this class for govuk-button before, can I ask what it does?

Reply:
Changed to the standard govuk button class